db/sql/rounds/insertRoundInformationWithRemovingDelegate.sql

```diff
@@ -0,0 +1,13 @@
+/*
+  DESCRIPTION: ?
+
+  PARAMETERS: ?
+*/
+
+INSERT INTO mem_round
```

Comment:
This should be just one SQL file, for `insertRoundInformationWithRemovingDelegate` and for `insertRoundInformationWithAddingDelegate`. They only differ in a `-` - minus sign.